build.xml

```diff
@@ -478,7 +478,11 @@
           <not><available file="${downloadsdir}/solr-${solr_version}.tgz" /></not>
            <then>
             <mkdir dir="${downloadsdir}" />
-            <httpget url="https://archive.apache.org/dist/solr/solr/${solr_version}/solr-${solr_version}.tgz" dir="${downloadsdir}" />
+            <exec executable="php" checkreturn="true" passthru="true">
+              <arg line="${srcdir}/public/index.php util/download https://archive.apache.org/dist/solr/solr/${solr_version}/solr-${solr_version}.tgz downloads/solr-${solr_version}.tgz" />
```

Comment:
You should use ${downloadsdir} here, shouldn't you?

Reply:
Oops, yes, that's the whole point... :)